poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.5.0 and should not be changed by hand.
```

Comment:
could you also bump the version of mex-common in the toml please?

Reply:
you marked this resolved, but i don't see this in the diff

Reply:
I am not sure why its not popping up in the diff, but mex-common version in pyproject.toml is 0.11.0 https://github.com/robert-koch-institut/mex-common/blob/100b4c54e8e96cc2cb59169b31594dbc0a08646a/pyproject.toml#L3C19-L3C19

and lock file also says its created by poetry 1.5.1